src/formatter/helpers/keyword_type.ts

```diff
@@ -1,6 +1,7 @@
 import _ from 'lodash'
 import Gherkin from 'gherkin'
 import { doesHaveValue } from '../../value_checker'
+import Dialect from 'gherkin/dist/src/Dialect'
```

Comment:
Is there no other way to access this without reaching into Gherkin? If not, lets open an issue in cucumber if that doesn't exist already

Reply:
Cool. Thanks for creating an upstream PR!